lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -628,6 +628,14 @@ ifeq "$(SWIFT_CXX_INTEROP)" "1"
 	SWIFTFLAGS += -Xcc -std=c++17
 endif
 
+#----------------------------------------------------------------------
+# Check if we should compile in Swift embedded mode
+#----------------------------------------------------------------------
+ifeq "$(SWIFT_EMBEDDED_MODE)" "1"
+	SWIFTFLAGS += -gdwarf-types -enable-experimental-feature Embedded -Xfrontend -disable-objc-interop
```

Comment:
Should -enable-experimental-feature Embedded imply -gdwarf-types?

Reply:
I guess it shouldn't turn on debug info if it's off

Reply:
What I'm trying to say is: The swift-driver should turn `-g -enable-experimental-feature Embedded` into `-gdwarf-types -enable-experimental-feature Embedded`. Feel free to make this a separate commit